src/app/containers/Include/index.jsx

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/no-danger */
+import React from 'react';
+import { string } from 'prop-types';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+import useToggle from '../Toggle/useToggle';
+
+const decodeHTML = str => {
+  const replacedParts = {
+    '&quot;': '"',
+    '&#39;': "'",
+    '&amp;': '&',
+    '&lt;': '<',
+    '&gt;': '>',
+  };
+  const replacementsRegex = new RegExp(
+    Object.keys(replacedParts).join('|'),
+    'gi',
+  );
+  return str.replace(replacementsRegex, match => replacedParts[match]);
+};
+
+const IncludeContainer = ({ html, type }) => {
+  const { enabled } = useToggle('include');
+
+  const supportedTypes = {
+    idt2: 'idt2',
+    vj: 'vj',
+  };
+
+  const shouldNotRenderInclude = !enabled || !html || !supportedTypes[type];
+
+  if (shouldNotRenderInclude) {
+    return null;
+  }
+
+  return (
+    <GridItemConstrainedMedium>
+      <div
+        suppressHydrationWarning
```

Comment:
Investigate if this is indeed necessary - are we rendering different things on the server than the client?

Reply:
this is needed as we're still decoding the markup

Reply:
The rest of this PR looks good, but I'd still like some clarification on this.

What happens if hydration is delayed or doesn't occur? Is the Include or page broken? Hydration could be delayed or not occur for multiple reasons. E.G. If the user is on a slow connection and client-side JS hasn't executed yet, or JS is disabled.